token-group/js/package.json

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "@solana/spl-token-group",
+    "description": "SPL Token Group Interface JS API",
+    "version": "0.0.1",
+    "author": "Solana Labs Maintainers <maintainers@solanalabs.com>",
+    "repository": "https://github.com/solana-labs/solana-program-library",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "engines": {
+        "node": ">=16"
+    },
+    "files": [
+        "lib",
+        "src",
+        "LICENSE",
+        "README.md"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "./lib/cjs/index.js",
+    "module": "./lib/esm/index.js",
+    "types": "./lib/types/index.d.ts",
+    "exports": {
+        "types": "./lib/types/index.d.ts",
+        "require": "./lib/cjs/index.js",
+        "import": "./lib/esm/index.js"
+    },
+    "scripts": {
+        "build": "tsc --build --verbose tsconfig.all.json",
+        "clean": "shx rm -rf lib **/*.tsbuildinfo || true",
+        "deploy": "npm run deploy:docs",
+        "deploy:docs": "npm run docs && gh-pages --dest token-group/js --dist docs --dotfiles",
+        "docs": "shx rm -rf docs && typedoc && shx cp .nojekyll docs/",
+        "fmt": "prettier --write '{*,**/*}.{ts,tsx,js,jsx,json}'",
+        "lint": "prettier --check '{*,**/*}.{ts,tsx,js,jsx,json}' && eslint --max-warnings 0 .",
+        "lint:fix": "npm run fmt && eslint --fix .",
+        "nuke": "shx rm -rf node_modules package-lock.json || true",
+        "postbuild": "shx echo '{ \"type\": \"commonjs\" }' > lib/cjs/package.json",
+        "reinstall": "npm run nuke && npm install",
+        "release": "npm run clean && npm run build",
+        "test": "mocha test",
+        "watch": "tsc --build --verbose --watch tsconfig.all.json"
+    },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.87.6"
+    },
+    "dependencies": {
+        "@solana/codecs-core": "2.0.0-experimental.9741939",
+        "@solana/codecs-data-structures": "2.0.0-experimental.9741939",
+        "@solana/codecs-numbers": "2.0.0-experimental.9741939",
+        "@solana/codecs-strings": "2.0.0-experimental.9741939",
+        "@solana/options": "2.0.0-experimental.9741939",
+        "@solana/spl-type-length-value": "0.1.0"
+    },
```

Comment:
Sorry, I just noticed I don't think `@solana/codecs-core` is being used in this library. Can we cut it?